src/odemis/acq/stitching/_weaver.py

```diff
@@ -47,6 +46,9 @@ def __init__(self, adjust_brightness=False):
         """
         self.tiles = []
         self.adjust_brt = adjust_brightness
+        self.tbbx_px = None
+        self.gbbx_px = None
+        self.gbbx_phy = None
```

Comment:
A short explanation on what these abbreviation mean would be nice.

Reply:
fixed